tests/publiccloud/run_ltp.pm

```diff
@@ -242,7 +242,7 @@ sub run {
 
     $self->dump_kernel_config($instance);
     record_info('LTP START', 'Command launch');
-    script_run($cmd_run_ltp, timeout => get_var('LTP_TIMEOUT', 30 * 60));
+    assert_script_run($cmd_run_ltp, timeout => get_var('LTP_TIMEOUT', 30 * 60));
```

Comment:
Preferable to report_info and store the result, then "die" after proper  elaborations completed.